controls/V-73763.rb

```diff
@@ -48,11 +49,12 @@
   Deny log on as a batch job to include the following:
 
   Domain Systems Only:
-  - Enterprise Admins Group
-  - Domain Admins Group
+  - Enterprise Admins group 
+  - Domain Admins group 
 
   All Systems:
-  - Guests Group"
+  - Guests group 
```

Comment:
Missing the end double-quotes - which seems to be leading to the following code to be treated as a string